.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,45 @@
+<!--
+Allo' allo'! 
+Thanks for taking your time to submit an issue.
+Please keep in mind, that Github issues are meant to be used for reporting bugs and to
+request new features.
+
+Use Stack Overflow for support: http://stackoverflow.com/questions/tagged/yeoman
+Head over to one of our Gitter rooms (https://gitter.im/yeoman/home) and ask for help if you’re unsure if you ran into a bug or if you have any other question.
+
+You would like to report a bug?
+Use the search feature to ensure that the bug hasn't been reported before.
+Please ensure to provide the following information to make sure we have all we need to address your issue.
+-->
+
+## Type of issue
+
+<!-- Feature request or bug -->
+
+<!-- Please delete the rest of the template in case of a feature request -->
+
+----
+
+## My environment
+
+* *OS version/details*: `eg. Windows 10 64-bit`
+* *Node version:* `x.x.x`
+* *npm version:* `x.x.x`
+* *Name of used Yeoman module*: `eg. yo`
+* *Version of Yeoman module:* `x.x.x`
```

Comment:
We could make the two lines above repo specific.

Remove: 
+ *Name of used Yeoman module*: `eg. yo`

And replace:
* *Version of Yeoman module:* `x.x.x`

with: 
* *Version of yo:* `x.x.x`